src/components/text-input/TextInput.constants.ts

```diff
@@ -0,0 +1,43 @@
+import { TIconGlyph } from '../../foundations/icon';
+
+import { TTextInputSizeToken, TTextInput } from './TextInput.types';
+
+const DEFAULT_TEXT_INPUT_SIZE: TTextInputSizeToken = 'md';
+
+const TEXT_INPUT_SIZES: TTextInputSizeToken[] = ['sm', 'md', 'lg'];
+
+const DEFAULT_LEADING_ICON: TIconGlyph = 'magnify';
```

Comment:
nit: only use default values for icons in the story ... we should not use them in the default component  implementation

Reply:
I am not passing them in the component they are just declared in the constants file for clarity. Then I pass the constant directly into the story

Reply:
Yep... I did see that. Just wanted to be sure here. 👍🏻